contracts/src/v0.8/automation/upkeeps/LinkAvailableBalanceMonitor.sol

```diff
@@ -216,37 +236,50 @@ contract LinkAvailableBalanceMonitor is ConfirmedOwner, Pausable, AutomationComp
   /// @param payee the address to pay
   function withdraw(uint256 amount, address payable payee) external onlyOwner {
     require(payee != address(0), "LinkAvailableBalanceMonitor: invalid payee address");
-    LINK_TOKEN.transfer(payee, amount);
+    i_linkToken.transfer(payee, amount);
     emit FundsWithdrawn(amount, payee);
   }
 
-  /// @notice Sets the top up amount
-  function setTopUpAmount(uint256 topUpAmount) external onlyOwner returns (uint256) {
-    require(topUpAmount > 0, "LinkAvailableBalanceMonitor: invalid linkTokenAddress");
-    return s_topUpAmount = topUpAmount;
+  /// @notice Sets the minimum balance for the given target address
+  function setMinBalance(address target, uint96 minBalance) external onlyOwner {
+    require(target != address(0), "LinkAvailableBalanceMonitor: invalid target address");
+    require(minBalance > 0, "LinkAvailableBalanceMonitor: invalid minBalance");
+    if (!s_targets[target].isActive) {
+      revert InvalidWatchList();
+    }
+    uint256 oldBalance = s_targets[target].minBalance;
+    s_targets[target].minBalance = minBalance;
+    emit BalanceUpdated(target, oldBalance, minBalance);
   }
 
   /// @notice Sets the minimum balance for the given target address
-  function setMinBalance(address target, uint256 minBalance) external onlyOwner returns (uint256) {
-    require(minBalance > 0, "LinkAvailableBalanceMonitor: invalid minBalance");
-    (bool exists, uint256 prevMinBalance) = s_watchList.tryGet(target);
-    if (!exists) {
+  function setTopUpAmount(address target, uint96 topUpAmount) external onlyOwner {
+    require(target != address(0), "LinkAvailableBalanceMonitor: invalid target address");
+    require(topUpAmount > 0, "LinkAvailableBalanceMonitor: invalid topUpAmount");
```

Comment:
`topUpAmount` can't be <0 because it's a uint96. Better to just do a == 0 check

Reply:
This check is that the topUp has to be > 0.

Reply:
sorry, I misread!